Makefile

```diff
@@ -136,6 +136,9 @@ perms-table:
 .PHONY: gen
 gen: cleangen proto api cli perms-table fmt copywrite
 
+.PHONY: gen-offline
+gen-offline: cleangen protobuild api cli perms-table fmt copywrite
+
```

Comment:
this definitely shouldn't be in this pr, but makes it easier for when you're working without internet